### PR TITLE
Introduction dir translation

### PR DESCRIPTION
**Pull request by imlidian**

### CODE REVIEW

File: java-chassis-reference/en_US/start/architecture.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 
-1.编程模型和通信模型分离，不同的编程模型可以灵活组合不同的通信模型。应用开发者在开发阶段只关注接口开发，部署阶段灵活切换通信方式；支持legacy系统的切换，legacy系统只需要修改服务发布的配置文件（或者annotation），而不需要修改代码。
+Currently, applications can be developed in Spring MVC, JAX-RS, or transparent RPC mode.
 
-现阶段支持SpringMVC、JAX-RS和透明RPC三种开发方式。
+2.To introduce the code-first approach. This allows cross-language communication through API definition-based microservice development and supports the development of software tool chains(such as code generated based on API definitions and API definitions generated based on code) to build a complete development ecosystem.
 
-2.内建API-first支持。通过契约规范化微服务开发，实现跨语言的通信，并支持配套的软件工具链（契约生成代码、代码生成契约等）开发，构建完整的开发生态。
+3.To define common microservice running model, encapsulating fault tolerance methods of microservices used from discovery to interaction, The running model can be customized or extended.
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/architecture.md]: interaction -> invocation

File: java-chassis-reference/en_US/start/architecture.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
-
-| 类型 | artifact id | 是否可选 | 功能说明 |
-| :--- | :--- | :--- | :--- |
-| 编程模型 | provider-pojo | 是 | 提供RPC开发模式 |
-| 编程模型 | provider-jaxrs | 是 | 提供JAX RS开发模式 |
-| 编程模型 | provider-springmvc | 是 | 提供Spring MVC开发模式 |
-| 通信模型 | transport-rest-vertx | 是 | 运行于HTTP之上的开发框架，不依赖WEB容器，应用打包为可执行jar |
-| 通信模型 | transport-rest-servlet | 是 | 运行于WEB容器的开发框架，应用打包为war包 |
-| 通信模型 | transport-highway | 是 | 提供高性能的私有通信协议，仅用于java之间互通。 |
-| 运行模型 | handler-loadbalance | 是 | 负载均衡模块。提供各种路由策略和配置方法。一般客户端使用。 |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/architecture.md]: 一般客户端使用 -> Consumer端使用，请对应修改下面的英文

File: java-chassis-reference/en_US/start/architecture.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+| Programming model      | provider-springmvc     | Yes              | Provides the Spring MVC development mode. |
+| Communication on model | transport-rest-vertx   | Yes              | A development framework running over HTTP, it does not depend on Web containers. Applications are packaged as executable .jar files. |
+| Communication on model | transport-rest-servlet | Yes              | A development framework running on Web container. Applications are packaged as WAR files. |
+| Communication on model | transport-highway      | Yes              | Provides high-performance private communication protocols for Java communication. |
+| Running model          | handler-loadbalance    | Yes              | A load balancing module that provides various routing policies and configurations. It is usually used for clients. |
+| Running model          | handler-bizkeeper      | Yes              | Provides service governance functions, such as isolation, fallbreak, and fault tolerance. |
+| Running model          | handler-tracing        | Yes              | Provides the capability of developing and managing TCC transactions. |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/architecture.md]: Provides the capability of developing and managing TCC transactions.
和TCC没有关系，Provides the capability of  tracing for integration with Monitor system, also can  self make Span

File: java-chassis-reference/en_US/start/development-environment.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
-# 安装本地开发环境
+# Prepare the local development environment
 
-应用开发环境所需安装的工具包括JDK、Maven、Eclipse 和 IDEA 。如果已经安装了这些开发工具，请跳过本节。
+The tools required to install the development environment include JDK, Maven, Eclipse, and IDEA. If you already have these development tools installed, skip this section.
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/development-environment.md]: The tools required to install the development environment include JDK, Maven, Eclipse, and IDEA ->
JDK, Maven, Eclipse, and IDEA is required.

File: java-chassis-reference/en_US/start/development-environment.md [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
 
-## JDK约束及安装步骤
+## JDK version and installation steps
 
-1.JDK版本
+1.JDK version
 
-JDK版本要求1.8以上。
+The JDK version requires 1.8 or more.
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/development-environment.md]: more -> higher

File: java-chassis-reference/en_US/start/development-environment.md [hunk 6]
```diff
@@ -27,33 +27,33 @@ C:\Users\h00407576> java -version
  Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
 ```
 
-## Maven安装步骤
+## Maven installation steps
 
-Maven是一款集项目管理、代码编译、工程管理等等能力于一体的开发工具。
+Maven is a development tool that integrates project management, code compilation, project management and more.
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/development-environment.md]: project management 重复

File: java-chassis-reference/en_US/start/development-environment.md [hunk 7]
```diff
@@ -64,9 +64,9 @@ f. （可选）要想项目能够快速的下载各种依赖，建议配置下
   </mirror>
 ```
 
-g.结果验证
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/development-environment.md]: 建议改成安装验证

File: java-chassis-reference/en_US/start/development-environment.md [hunk 8]
```diff
@@ -79,46 +79,46 @@ C:\Users\h00407576>mvn -version
...
 
-a. 在官方网站下载 IDEA 安装包，收费版或者社区版的按个人需求。
+a. Download the IDEA installation package on the official website, the paid version or the community version according to individual needs.
 
-b. 设置编码格式都为 utf-8。
+b. Set the encoding format to utf-8.
 
-打开IDEA，选择 File -> Settings -> Editor -> File Encoding
-把 project Encoding 和 default encoding for properties files 改为 utf-8。
+Open IDEA and select File -> Settings -> Editor -> File Encoding
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/development-environment.md]: 建议使用`project Encoding`和`default encoding for properties files`这种高亮

File: java-chassis-reference/en_US/start/first-sample.md [hunk 9]
```diff
@@ -1,18 +1,17 @@
-# 开发第一个微服务
+# Develop the first microservice
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/first-sample.md]: 这一段你的内容需要更新，开发微服务的方式有两种：
1. 使用Archetype，Archetype的说明在https://github.com/apache/incubator-servicecomb-java-chassis/blob/master/archetypes/README.md
2. 使用start.servicecomb.io

File: java-chassis-reference/en_US/start/terminology.md [hunk 10]
```diff
@@ -1,23 +1,20 @@
...
-| Circuit Breaker | Circuit Breaker | 熔断 | 微服务之间通常存在依赖关系，服务调用链路可能包含多个微服务，如果链路中一个或多个服务访问延迟过高，会导致入口服务的请求不断堆积，持续消耗更多的线程、io资源，最终由于资源累积使系统出现瓶颈，造成更多服务不可用，产生雪崩效应。熔断机制就是针对上述场景设计的，当某个目标服务响应缓慢或者有大量超时情况发生时，熔断该服务的调用，对于后续调用请求，不再继续调用目标服务，直接返回，快速释放资源，等到该目标服务情况好转再恢复调用。 |
-| Isolation | Isolation | 隔离 | 服务隔离是一种异常检测机制，常用的检测方法是请求超时、流量过大等。一般的设置参数包括超时时间、最大并发请求数等，当超过超时时间或最大并发请求数时，记录一次异常，并用于在熔断机制中计算错误率和错误请求数。 |
-| Service Mesh | Service Mesh | 服务网格 | 一种基础设施层服务。在微服务化的过程中，开发者需要解决应用运行在分布式网络中所引入的问题，例如容错，限流，负载均衡，注册发现，可监控等，Service Mesh作为L4/L7协议代理，为应用解决了微服务化后带来的问题。 |
-| legacy | legacy | 遗留系统 | 遗留系统是一个还在运行和使用，但已步入软件生命周期衰老期的软件系统。 |
-
-
-
+MicroServices | MicroServices |  微服务 | Microservices is a lightweight SOA architecture that is commonly used to describe a loosely coupled distributed architecture that is widely used in cloud applications and Internet applications. |
+| Provider | Provider | 服务提供者  | The service of the called party in the microservice invocation relationship. |
+| Consumer | Consumer | 服务消费者 | The service that invokes the initiator in the microservice invocation relationship. |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/terminology.md]: initiator这个词好像不太准确

File: java-chassis-reference/en_US/start/terminology.md [hunk 11]
```diff
@@ -1,23 +1,20 @@
...
+IAM | Identity and Access Management | 身份及权限管理 | Responsible for maintaining the management level, user, role, authorization relationship, user organization and other information in the PaaS system. And carry out authorization and authorization checks. |
+AK/SK | AK/SK Key | AK/SK密钥 |Access key/Secret key is a set of key pairs used for API authentication and access control. |
+| Service | Service | 服务 | A service is a description of a functional object that is accessed on demand. In the application model, the service is generally oriented to the application. The application usage service needs to subscribe to the service first, then bind the service and use it. In some business scenarios, it may also need to pay according to usage. |
+Load Balance | Load Balance | 负载均衡 | When an application accesses a microservice with multiple instances, it involves routing load balancing. Load balancing policies can be configured through configuration files to support multiple load balancing routing policies such as random, polling, session hold, and response time based weights. |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/terminology.md]: polling -> RoundRobin

File: java-chassis-reference/en_US/start/terminology.md [hunk 12]
```diff
@@ -1,23 +1,20 @@
...
+Service Degrade | Service Degrade | 降级 | Service downgrade mainly includes two strategies: masked downgrade and fault tolerant downgrade: shielded downgrade refers to the decision of the operation and maintenance personnel/developers when the trigger condition of the outside world reaches a certain critical value. Or a service is forced to downgrade. Fault-tolerant degradation means that when non-core services are unavailable, business logic can be released for faulty services to ensure the operation of core services. |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/terminology.md]: 这一段，masked downgrade，fault tolerant downgrade，shielded downgrade，感觉没对上

File: java-chassis-reference/en_US/start/terminology.md [hunk 13]
```diff
@@ -1,23 +1,20 @@
...
+Fault tolarance | Fault tolarance | 容错 | Fault Tolerance is a processing strategy in the scenario where an exception occurs when a consumer accesses a service. After an exception occurs, the service framework automatically selects a new service route to invoke. |
+Circuit Breaker | Circuit Breaker | 熔断 | There is usually a dependency between microservices. The service invocation link may contain multiple microservices. If one or more service access delays in the link are too high, the request for the portal service will continue. Stacking, continue to consume more threads, io resources, and eventually the system bottleneck due to resource accumulation, resulting in more services unavailable, resulting in avalanche effect. The fuse mechanism is designed for the above scenario. When a target service responds slowly or a large number of timeouts occur, the service call is blown. For subsequent call requests, the target service is no longer called, directly returned, and resources are quickly released. The target service situation is improved and the call is resumed. |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/terminology.md]: Stacking？这段建议理解中心手动翻译，简化细节

File: java-chassis-reference/en_US/start/terminology.md [hunk 14]
```diff
@@ -1,23 +1,20 @@
...
+Isolation | Isolation | 隔离 | Service isolation is an anomaly detection mechanism. Common detection methods are request timeout, excessive traffic, and so on. The general setting parameters include the timeout period, the maximum number of concurrent requests, etc. When the timeout period or the maximum number of concurrent requests is exceeded, an exception is recorded and used to calculate the error rate and the number of error requests in the fuse mechanism. |
+Service Mesh | Service Mesh | 网路服务 | An infrastructure layer service. In the process of micro-service, developers need to solve the problems introduced by applications running in distributed networks, such as fault tolerance, current limiting, load balancing, registration discovery, monitoring, etc. Service Mesh acts as the L4/L7 protocol agent. The application solves the problems caused by micro-services. |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/terminology.md]: 服务网格是正确的，为什么要改为网路服务？

File: java-chassis-reference/en_US/start/terminology.md [hunk 15]
```diff
@@ -1,23 +1,20 @@
...
+Legacy | Legacy | 遗留系统 | A legacy system is a software system that is still running and in use, but has entered the aging phase of the software lifecycle. |
```
Comment (zhengyangyong) [inline java-chassis-reference/en_US/start/terminology.md]: but has entered the aging phase of the software lifecycle. ->
这一段可以不用扣原文，建议改为架构已落后的系统

**Comment by imlidian** [comment]
all done

File: java-chassis-reference/en_US/start/first-sample.md (hunk 9 above)
Comment (WillemJiang) [reply java-chassis-reference/en_US/start/first-sample.md]: @zhengyangyong  Could you update the document later? I suggest we just introduce start.servicecomb.io.

**Pull request merged**